opm/io/eclipse/OutputStream.hpp

```diff
@@ -276,6 +276,87 @@ namespace Opm { namespace EclIO { namespace OutputStream {
                        const std::vector<T>& data);
     };
 
+    /// File manager for RFT output streams
+    class RFT
+    {
+    public:
+        struct OpenExisting { bool set; };
+
+        /// Constructor.
+        ///
+        /// Opens file stream for writing.
+        ///
+        /// \param[in] rset Output directory and base name of output stream.
+        ///
+        /// \param[in] fmt Whether or not to create formatted output files.
+        ///
+        /// \param[in] existing Whether or not to open an existing output file.
+        explicit RFT(const ResultSet&    rset,
```

Comment:
explicit is implied.

Reply:
https://abseil.io/tips/142

Reply:
alright, learned something new.

Reply:
Interesting, I'm undecided on applying this tip though, need to think about it.